ner/scripts/train.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
 #
-# This is a simple training script. It takes two arguments, the first specifies the directory
-# containing the training data, the second specifies the directory containing the test data, 
-# and the last argument specifies the config file.
+# This is a simple training script. It takes 4 arguments, the first specifies the directory
+# containing the training data, the second specifies the directory containing the test data,
+# the third specifies the input file format, and the last argument specifies the config file.
```

Comment:
Would be nice if you say what are the example options for the "format" input.

Reply:
Here and in the other scripts.

Reply:
Yes... these are spelled out in NerTagger.java, and in the usage document. I was trying to avoid duplication of documentation, which is bound to get out of date.

Reply:
Alright; good point. I'm convinced.